packages/react-native/src/primitives/RadioGroup/types.ts

```diff
@@ -0,0 +1,26 @@
+import { TextStyle, ViewStyle } from 'react-native';
+import { RadioProps } from '../Radio';
+
+export type Direction = 'vertical' | 'horizontal';
+
+export interface RadioGroupProps<T>
+  extends Pick<
+    RadioProps<T>,
+    | 'accessibilityRole'
+    | 'disabled'
+    | 'label'
+    | 'labelPosition'
+    | 'labelStyle'
+    | 'onChange'
+    | 'size'
+    | 'value'
```

Comment:
Why not just omit the props we don't want?

Reply:
`RadioProps` extends from `PressableProps`

Reply:
Ah, makes sense now, ty